site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/service/SiteServiceImpl.java

```diff
@@ -1,14 +1,16 @@
 package uk.ac.ox.ndph.mts.site_service.service;
 
+import org.hl7.fhir.r4.model.Organization;
```

Comment:
Not sure about this one - should the service layer use a repository level entity?
I would think that the service layer should work with the business level entity type which is **Site**. WDYT?

Reply:
Addressed as discussed